src/ert/_c_wrappers/enkf/queue_config.py

```diff
@@ -161,6 +161,16 @@ def lsf_server(self):
     def num_cpu(self):
         return self._get_num_cpu()
 
+    def __repr__(self):
+        return (
+            "QueueConfig(config_dict={"
+            f"{ConfigKeys.JOB_SCRIPT}: {self.job_script}, "
+            f"{ConfigKeys.MAX_SUBMIT}: {self.max_submit}, "
+            f"{ConfigKeys.NUM_CPU}: {self.num_cpu}, "
+            f"{ConfigKeys.QUEUE_SYSTEM}: {self.queue_system}, "
+            "})"
```

Comment:
no single quotes in this representation?

Reply:
fixed